### ISSUE TITLE
Investigate broken "latest" links in regards to recent phet-io maintenance release

### ISSUE THREAD

**Issue by zepumph** [phetsims/perennial#99]
@mattpen found that the latest links were broken for "chipper 1.0" simulations deployed as part of https://github.com/phetsims/phet-io/issues/1329 (the list of simulations is https://github.com/phetsims/phet-io/issues/1329#issuecomment-407122396).

The problem was that the phet-io simulations, although when developed were deployed with `-phetio` in the version directory, had that omitted in the recent maintenance release for a scenery patch.

After further investigation, we found in the logs that build-server received the request as `api: "2.0"` and with the `-phetio` omitted in the version string. This means that the request was sent as such. We also noted that the build server does have the logic built in to handle chipper 1.0 properly if it received an `api: "1.0"` request. Basically this means that the build server is set up as it should be to support chipper 1.0 and 2.0 builds.

We think that the problem is a result of conducting the production deploy for these chipper 1.0 simulations from master of perennial, which doesn't have any logic to send api 1.0 requests the build server, nor does it add in `-phetio` to the version string. Perhaps if the deploys had been called in chipper (so in the sim repo) rather than from perennial, then the proper version string would have been upheld.

**How To Proceed**:
Although this has broken the latest links on the phet-io website. @mattpen and I are ok leaving them broken for this maintenance. This is because no one is using the latest links, and the problem should be fixed the next time we release a sim.

Tagging @jonathanolson so he is aware of the problem, and potentially so that he can comment if the above findings seem incorrect to him. The only action needed is for him to read the issue. 

On a side note, I wonder how much of the automated build process that @jonathanolson used for this maintenance release depended on master of perennial, vs using chipper to deploy for phet-io simulations using chipper 1.0.

**Comment by jonathanolson** [comment]
This seems incorrect to me. The build-server would presumably inspect the chipper SHA's package.json to determine whether it is for chipper 2.0 or not, and that should be fundamentally orthogonal to what api version is used for the request.

Presumably the build server logic should detect this, and create the proper directory name.

**Comment by mattpen** [comment]
@jonathanolson and @zepumph and I discussed on a call.

The main problem is a regression where chipper 1.0 phet-io brand deploys used to send versions like `x.x.x-phetio`, but are now being sent as `x.x.x`.  This caused the wrappers to fail, so we changed them manually, but this manual change caused the latest links to fail.  The build server will now need to assume that it must append `-phetio` to the dir name for chipper 1.0 phet-io brand deploys.  I'll make this change and deploy it ASAP.
